src/System.ComponentModel.EventBasedAsync/tests/AsyncOperationTests.cs

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.ComponentModel;
+using System.Threading;
+using Xunit;
+
+namespace System.ComponentModel.EventBasedAsync
+{
+    public class AsyncOperationTests
+    {
+        private const int k_spinTimeoutSeconds = 20;
+
+        [Fact]
+        public static void Noop()
+        {
+            // Test that a simple AsyncOperation can be dispatched and completed via AsyncOperationManager
+            var operation = new TestAsyncOperation(op => {});
+            operation.Wait();
```

Comment:
I'll change it to wrapped in Task.Run, and apply same concept to other tests as well.